backend/rebutify/settings.py

```diff
@@ -191,3 +191,10 @@
 EMAIL_USE_TLS = bool(os.getenv("EMAIL_USE_TLS", True))
 
 EMAIL_FROM = os.getenv("EMAIL_FROM", "")
+
+DJOSER = {
+    "ACTIVATION_URL": "activate/{uid}/{token}",
```

Comment:
@Monstarrrr this is the line you want to change if you want to change how the link in emails looks.